packages/react-positioning/src/types.ts

```diff
@@ -0,0 +1,190 @@
+import * as PopperJs from '@popperjs/core';
+import * as React from 'react';
```

Comment:
Will exporting everything from this file leak Popper types ? @layershifter

Reply:
I think that it's safe after https://github.com/microsoft/fluentui/pull/17339/files#r590585228.